### ISSUE TITLE
Devise a stable tracking mechanism

### ISSUE THREAD

**Issue by Werkov** [JankaSvK/thesis#9]
- define what stable is
- likely depends on #7 (reproducible results) and #8 ((semi-)automated evaluation)
- perhaps compare multiple approaches as per definition above and pick the "best" one
  - as a side effect this may give lots of data which can be broken down and analyzed in the thesis
    - this is not part of this issue
